docs/userstories/increase_pvc_size_jh.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Increase PVC size in JupyterHub
+
+    Background:
+        Given I am a user of MOC-CNV
+        * I have access to JupyteHub on MOC-CNV
+
+    Scenario: I use the default PVC
+
+        This scenario is based on [operate-first/apps#243](https://github.com/operate-first/apps/pull/243) pull request.
```

Comment:
The link doesn't form when using a code block, maybe just leave link as is without the `[]()` syntax ?